### PR TITLE
Updated MessageStubBuilder & LwdpMessageFieldsResolver

### PR DESCRIPTION
**Pull request by xHacka**

### CODE REVIEW

File: th2_data_services/data_source/lwdp/resolver.py [hunk 1]
```diff
@@ -61,12 +61,15 @@ def get_attached_messages_ids(event):
     def get_body(event):
         return event[http_event_struct.BODY]
 
-
 class LwdpMessageFieldsResolver(MessageFieldsResolver):
     @staticmethod
     def get_subsequence(message):
-        raise NotImplementedError
+        return message[http_message_struct.BODY]["metadata"][http_message_struct.SUBSEQUENCE]
```
[the remark this answers is not in the document]
Comment (ConnectDIY) [reply th2_data_services/data_source/lwdp/resolver.py:67]: Moved to SubMessageFieldResolver

File: th2_data_services/data_source/lwdp/resolver.py [hunk 2]
```diff
@@ -61,12 +61,15 @@ def get_attached_messages_ids(event):
...
 
+    @staticmethod
+    def get_protocol(message):
```
[the remark this answers is not in the document]
Comment (ConnectDIY) [reply th2_data_services/data_source/lwdp/resolver.py:70]: Moved to SubMessageFieldResolver

File: th2_data_services/data_source/lwdp/resolver.py [hunk 3]
```diff
@@ -77,23 +80,19 @@ def get_session_id(message):
 
     @staticmethod
     def get_type(message):
-        return message[http_message_struct.MESSAGE_TYPE]
+        return message[http_message_struct.BODY]["metadata"][http_message_struct.MESSAGE_TYPE]
```
Comment (ConnectDIY) [inline th2_data_services/data_source/lwdp/resolver.py:83]: Moved to SubMessageFieldResolver

here should be Notimplemented Error

File: th2_data_services/data_source/lwdp/resolver.py [hunk 4]
```diff
@@ -117,4 +116,4 @@ def get_attached_event_ids(message):
 
     @staticmethod
     def get_fields(message):
-        return message[http_message_struct.BODY]["fields"]
+        return message[http_message_struct.BODY]["fields"]
```
Comment (ConnectDIY) [inline th2_data_services/data_source/lwdp/resolver.py:119]: Moved to SubMessageFieldResolver

File: th2_data_services/data_source/lwdp/stub_builder.py [hunk 5]
```diff
@@ -111,7 +111,6 @@ def template(self) -> dict:
         return {
             self.message_fields.DIRECTION: None,
             self.message_fields.SESSION_ID: BrokenMessage(),
-            self.message_fields.MESSAGE_TYPE: BrokenMessage(),
```
Comment (ConnectDIY) [inline th2_data_services/data_source/lwdp/stub_builder.py:114]: Stub builder should return the message that looks like the message from LwDP. 
I didn't check, just make sure that it's so

**Review by ConnectDIY — CHANGES_REQUESTED** [review]

File: th2_data_services/data_source/lwdp/stub_builder.py (hunk 5 above)
Comment (xHacka) [reply th2_data_services/data_source/lwdp/stub_builder.py:114]: v3 structure doesn't have `MESSAGE_TYPE` in top level dictionary, that's why I removed it

**Review by ConnectDIY — APPROVED** [review]

**Pull request merged**